scripts/qa_crowbarsetup.sh

```diff
@@ -3331,7 +3327,7 @@ function oncontroller_upload_defcore
     # run only the specified tests
     tempest run --whitelist-file defcore.txt
     source /root/.openrc
-    ostestr last --subunit --no-pretty | subunit-2to1 > tempest.subunit.log
+    testr last --subunit > tempest.subunit.log
     test -d refstack-client || safely git clone https://github.com/openstack/refstack-client
     yes | refstack-client/refstack-client upload-subunit --keystone-endpoint $OS_AUTH_URL tempest.subunit.log
```

Comment:
I think this might be failed because refstack-client doesn't support v2 reports. Are you thinking of pushing a following patch? Or I just miss something?

Reply:
Correct. I was planning to fix it in a next PR.

Reply:
thanks! cool.